### DISCUSSION TITLE
Stand-alone "portal" creation tool please

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5246] [Announcements]
There are two unintended features of this approach

1. "Portals" now work the same way for public users and operators
2. Since there's no dedicated user involved, it's possible to move (or probably copy) the portal section of manage collection....

<img width="799" alt="Screen Shot 2020-06-19 at 10 57 38 AM" src="https://user-images.githubusercontent.com/5720791/85166144-b93f0e80-b21b-11ea-87e1-8eb7f2baae6f.png">

to a standalone app, which would let Operators customize their own portals (currently only Lam can do that), or even create arbitrary portals by selecting one or more collections (not even necessarily collections to which the user has access) and optionally providing CSS and/or customization parameters. That's likely to involve some social hurdles - please open a new issue if this is desirable.

_Originally posted by @dustymc in https://github.com/ArctosDB/arctos/issues/2683#issuecomment-646800671_

Yes, this is desirable.

**Comment by Jegelewicz** [comment]
Also, it would be nice if the number of records displayed at the top of the page reflected the number of records in the selected collections. So if I select http://arctos.database.museum/nmmnh_all
this number would be 112,028

![image](https://user-images.githubusercontent.com/5725767/87439436-05c50200-c5ae-11ea-9de0-874b23e45449.png)

![image](https://user-images.githubusercontent.com/5725767/87439613-3a38be00-c5ae-11ea-8f04-b9e6e5d64806.png)

Or maybe we could have this:

Access to 1,024,314 records, currently searching 112,028

**Comment by lin-fred** [comment]
> Access to 1,024,314 records, currently searching 112,028

I like that

**Comment by dustymc** [comment]
I dislike everything about "portals" and there's a fully functional API - tentatively closing.

**Comment by Jegelewicz** [comment]
But we are supposed to be doing this?

![image](https://user-images.githubusercontent.com/5725767/154311269-06f21e3d-0bbc-4828-868c-e7f97eb6101a.png)

**Comment by dustymc** [comment]
Not sure how that's portals - https://github.com/ArctosDB/arctos/issues/1555 maybe - but happy to discuss.
